### ISSUE TITLE
Feature request: Dynamic VLAN

### ISSUE THREAD

**Issue by mbecks** [cucumber-tony/cucumber-frontend#400]
OpenWRT and hostapd support a feature called 'Dynamic VLAN', where clients are automatically put in different VLANs as defined by a RADIUS server. This is basically an extension for the existing 802.1x security mode. I think this would be a great feature for CT and should also be easy to implement.

s. https://wiki.openwrt.org/doc/howto/wireless.security.8021x

I believe there are at least two options that should be made available to the customer per network:
1. Enable / Disable support for dynamic VLAN
2. Set a default VLAN for clients that have no VLAN assigned by RADIUS (may be 'none' in order to isolate such clients)

**Comment by AndrewMargarit** [comment]
Good idea.
I'll have a look and see how we can do it.

**Comment by AndrewMargarit** [comment]
@mbecks have you ever tried to get a working AP in this scenario?

**Comment by mbecks** [comment]
Yes, I have. Do you need a working wireless config as an example?

**Comment by AndrewMargarit** [comment]
That would speed things up.
Is it just the wireless config? Or does it also include the network/firewall?

**Comment by mbecks** [comment]
Below is a working wireless and network config that uses dynamic VLAN. The device itself is configured to use VLAN 10 as the management VLAN, just to show how this can interact with dynamic VLAN.
Wireless clients can either be assigned to an existing VLAN or to a completely new one (which will then be created by hostapd automatically), depending on the VLAN ID assigned by RADIUS to each individual client.

[config.tar.gz](https://github.com/cucumber-tony/cucumber-frontend/files/592386/config.tar.gz)

**Comment by simonmorley** [comment]
@AndrewMargarit Can you implement this on your own?

**Comment by AndrewMargarit** [comment]
No.
This needs to go in the configs.
